### PR TITLE
[exporter/influxdbexporter] Add support for exporting to InfluxDB v1.x API

### PR DESCRIPTION
**Pull request by j-kap-t**
**Description:** <Describe what has changed.>

Add support for exporting to InfluxDB v1.x API

**Link to tracking Issue:** <Issue number if applicable> n/a

**Testing:** <Describe what testing was performed and which tests were added.> Tested exporting to InfluxDB v1.x

**Documentation:** <Describe the documentation added.> Added config options to docs

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
Thanks for the contribution @j-kap-t, please sign the CLA so this can get reviewed

**Review by codeboten — COMMENTED** [review]
Pinging @jacobmarble as code owner

**Comment by j-kap-t** [comment]
@jacobmarble Any updates on this?

**Review by jacobmarble — APPROVED** [review]

**Comment by j-kap-t** [comment]
Thanks @jacobmarble! 

@codeboten What are the next steps to get this merged? I just realized the changelog validation failed due to lack of issue specified. Do I need to create a new issue and update the changelog file? Thanks!

File: .chloggen/feat_influxdb-1x.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: influxdbexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add support for exporting to InfluxDB v1.X API
+
+# One or more tracking issues related to the change
+issues: []
```
Comment (codeboten) [inline .chloggen/feat_influxdb-1x.yaml:11]:
```suggestion
issues: [16042]
```

**Comment by codeboten** [comment]
I just added the PR number to the changelog, should be good to go

**Review by codeboten — APPROVED** [review]

**Pull request merged**